.github/workflows/release-server-to-docker.yml

```diff
@@ -37,20 +46,28 @@ jobs:
           context: .
           push: true
           platforms: linux/amd64
-          tags: mitre/heimdall2:release-latest,mitre/heimdall2:${{ steps.format-tag.outputs.replaced }}
+          tags: mitre/heimdall2:release-latest,mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} # mainline can stay at 'latest' and only update the digest instead of tags and digest
       - name: Get Docker SHA
         shell: bash
         id: get-docker-sha
-        run: echo "DOCKER_SHA=$(docker pull mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} > /dev/null 2>&1 && docker inspect --format='{{index .RepoDigests 0}}' mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} | cut -d '@' -f 2)" >> $GITHUB_ENV
-      - name: Sophos Factory pipeline
-        uses: sophos/factory-run-pipeline@v2
-        with:
-          project_id: 601c48ad71a5df94e698a9ad
-          job_id: 644185303992f67e1d9412f2
-          api_token: ${{ secrets.SOPHOS_FACTORY_TOKEN }}
-          variables: |
-            {
-              "gitlabHost": "repo1.dso.mil",
-              "heimdallVersion": "${{ steps.format-tag.outputs.replaced }}",
-              "heimdallDockerURL": "docker://docker.io/mitre/heimdall2@${{ env.DOCKER_SHA }}"
-            }
+        run: |
+          echo "DOCKER_SHA=$(docker pull mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} > /dev/null 2>&1 && docker inspect --format='{{index .RepoDigests 0}}' mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} | cut -d '@' -f 2)" >> $GITHUB_ENV
+      - name: Make working directory for Iron Bank changes
+        run: mkdir ../ironbank_heimdall
+      - name: Clone and update local copy of Iron Bank repo to have the latest tags and digest
+        working-directory: ../ironbank_heimdall
+        run: |
+          git clone https://repo1.dso.mil/dsop/mitre/security-automation-framework/heimdall2.git .
+          git switch -c "${{ env.VERSION_UNDERSCORED }}" # swap to sha for mainline
+          yq e -i '.args.HEIMDALL_VERSION="${{ steps.format-tag.outputs.replaced }}" | .tags[0]="${{ steps.format-tag.outputs.replaced }}" | .labels."org.opencontainers.image.version"="${{ steps.format-tag.outputs.replaced }}" | .resources[1].tag="mitre/heimdall2:${{ steps.format-tag.outputs.replaced }}" | .resources[1].url="docker://docker.io/mitre/heimdall2@${{ env.DOCKER_SHA }}"' hardening_manifest.yaml
+          sed -i s/HEIMDALL_VERSION=\.\*/HEIMDALL_VERSION=${{ steps.format-tag.outputs.replaced }}/ Dockerfile
+          git diff
+          git add hardening_manifest.yaml Dockerfile
+          git -c "user.name=Automated Heimdall Release" -c "user.email=saf@groups.mitre.org" commit -s -m "Updating Heimdall to ${{ steps.format-tag.outputs.replaced }}"
```

Comment:
Do you still want to change the user.email? This seems appropriate to me.

Reply:
I thought it'd be more exact to have the releaser's id tied to the pr but you're right that we can probs just use the group email